components/src/dynamo/rl_admin/service.py

```
@@ -0,0 +1,434 @@
+
+
+# ══════════════════════════════════════════════════════════════════════
+# Main
+# ══════════════════════════════════════════════════════════════════════
+
+def main():
+    import uvicorn
+
+    parser = argparse.ArgumentParser(description="Dynamo RL Admin Service")
+    parser.add_argument("--port", type=int, default=8002, help="Port to listen on")
+    parser.add_argument("--host", type=str, default="0.0.0.0", help="Host to bind to")
```

Ruff (0.15.11)

[error] Possible binding to all interfaces

(S104)